Readme.md

```diff
@@ -10,14 +10,16 @@ Locally can run on regtest.
 
 ## Running locally
 
-Supposing to have [rust installed](https://www.rust-lang.org/tools/install) and a synced [bitcoin core](https://bitcoincore.org/en/download/) on mainnet with `txindex=1` and `rest=1` do:
+Supposing to have [rust installed](https://www.rust-lang.org/tools/install) >= 0.1.7 and a synced [bitcoin core](https://bitcoincore.org/en/download/) on mainnet with `txindex=1` and `rest=1` do:
```

Comment:
`0.1.7` is the version of what? Doesn't seem a rust version number

Reply:
in your other comment you mention `first` as being the highest req for the version, 
so you mean `1.66.0` https://doc.rust-lang.org/std/collections/struct.BTreeSet.html#method.first ?

Reply:
prev version cause it:
```
error[E0658]: use of unstable library feature 'map_first_last'                                                                                  
   --> src/threads/update_mempool_info.rs:256:38                                                                                                
    |                                                                   
256 |         mempool_fees.highest = rates.last().map(Into::into);                                                                              
    |                                      ^^^^                                                                                                 
    |                                                                   
    = note: see issue #62924 <https://github.com/rust-lang/rust/issues/62924> for more information                                                                                                                                                                                               

For more information about this error, try `rustc --explain E0658`.                                                                             
error: could not compile `fbbe` due to previous error                                                                                           
warning: build failed, waiting for other jobs to finish... 
```

Reply:
thanks!

If you feel it would be useful to add this version also here https://github.com/RCasatta/fbbe/blob/master/.github/workflows/test.yml#L41 so that is enforced by CI

Reply:
also, please squash the 2 commits changing the readme